includes/discovery/sensors/power/eltex-mes23xx.inc.php

```diff
@@ -19,46 +19,45 @@
  * @package    LibreNMS
  * @link       https://www.librenms.org
  *
- * @copyright  2022 Peca Nesovanovic
+ * @copyright  2025 Peca Nesovanovic
  *
  * @author     Peca Nesovanovic <peca.nesovanovic@sattrakt.com>
  */
-$divisor = 1000;
-$multiplier = 1;
-if ($pre_cache['eltex-mes23xx-poe']) {
-    foreach ($pre_cache['eltex-mes23xx-poe'] as $index => $data) {
+
+use LibreNMS\Util\Oid;
+
+$oids = SnmpQuery::hideMib()->walk('MARVELL-POE-MIB::rlPethPsePortPowerLimit')->table(2);
```

Comment:
walk can accept an array of oids that will have the same effect as your code here.

Reply:
yes, you are right